discohorts/pipeline.py

```diff
@@ -97,13 +102,15 @@ def get_patient_to_work_dir(patients, work_dirs):
                         check_call(command)
 
                 if ran_count % self.batch_size == 0:
```

Comment:
So grateful that you implemented this; have been scripting this over and over again for the previous runs :clap: